clients/java/signalr/src/test/java/com/microsoft/signalr/MockTransport.java

```diff
@@ -8,10 +8,21 @@
 import java.util.function.Consumer;
 
 class MockTransport implements Transport {
+    private static final String RECORD_SEPARATOR = "\u001e";
+
     private OnReceiveCallBack onReceiveCallBack;
     private ArrayList<String> sentMessages = new ArrayList<>();
     private String url;
     private Consumer<String> onClose;
+    private boolean ignorePings;
+
+    public MockTransport() {
+        this(false);
+    }
+
+    public MockTransport(boolean ignorePings) {
```

Comment:
Yeah, this is gonna conflict with the autoHandshake ctor that I added. We can add another ctor that takes two bool args but that might look weird.

Reply:
Could just add a setter as well